dist/templates/ovs-node.yaml.j2

```diff
@@ -0,0 +1,130 @@
+---
+# ovs-node
+# daemonset version 3
+# starts node daemons for ovs
+# it is run on all nodes
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovs-node
+  # namespace set up by install
+  namespace: ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the ovs networking components for all nodes.
+spec:
+  selector:
+    matchLabels:
+      app: ovs-node
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: ovs-node
+        name: ovs-node
+        component: network
+        type: infra
+        kubernetes.io/os: "linux"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      # Requires fairly broad permissions - ability to read all services and network functions as well
+      # as all pods.
+      serviceAccountName: ovn
+      hostNetwork: true
+      {{ "hostPID: true" if ovn_unprivileged_mode=="no" }}
```

Comment:
I _think_ this is still needed, since in privileged mode the CNI plugin does some direct ovs-ofctl/vsctl commands. Right?

Reply:
Honestly, I coped paste, but if it should run in the host instead of a pod, it makes sense to use hostPID

Reply:
it is needed.

Reply:
@aojea I don't think this is needed. The privileged/unprivileged mode is in the context of OVN CNI Shim and OVN CNI Server. If you want the the ovnkube-node (aka OVN CNI Server) to do all the Namespace related work, then you need to run in privileged mode. Otherwise, we use unprivileged mode.

As long as, the ovs-ofctl and ovs-vsctl calls use AF_UNIX sockets out of '/var/run/openvswitch' directory, then there is no need to run this with hostPID set to `true`